dotCMS/src/main/java/com/dotmarketing/db/listeners/DotListener.java

```diff
@@ -0,0 +1,15 @@
+package com.dotmarketing.db.listeners;
+
+interface DotListener extends Runnable {
+    
+    
+    public abstract String key();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary modifiers 'public abstract' on method 'key': the method is declared in an interface type](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5639225)